annotations/manifest_upload.py

```diff
@@ -76,10 +76,13 @@ def edit_manifest(file_path, col_types_dict):
     for column_name in df:
         if col_types_dict[column_name] == list:
             df[column_name] = df[column_name].astype(str)
-            df[column_name] = df[column_name].str.split(', ')
+            df[column_name] = df[column_name].str.strip()
+            df[column_name] = df[column_name].str.split(',')
```

Comment:
I think it would be better to do the whitespace trim after the column has been split.  For example, this would unfortunately still result in leading spaces after we split `df[column_name]`:

```python
>>> df = pd.DataFrame([["a,b,c"], ["d, e, f"]], columns=["foo"])
>>> df
       foo
0    a,b,c
1  d, e, f
>>> df['foo'].str.strip().str.split(',')
0      [a, b, c]
1    [d,  e,  f]
```

(Here, I've just combined lines 79 and 80 together into a single step). Notice how `e` and `f` have the leading spaces.

We need to add something after `split` so that the end result would look like this:

```python
>>> df['foo']
0    [a, b, c]
1    [d, e, f]
```

Hint: I would use the pandas function, `apply`.

Reply:
Ah great! Tom helped me. I think its fixed!

Reply:
Wooo, exactly how I would've done it 😊